src/server/identity/server/dex_web.go

```diff
@@ -60,6 +61,10 @@ var (
 // This is always /dex-assets in the docker image, but it can be overriden for testing
 var webDir = "/dex-assets"
 
+func WebDir(dir string) {
```

Comment:
@acohen4 I wanted to get your thoughts on if this is an okay way to access and change the path to '/dex-assets'

Reply:
feels unideal. WDYT about figuring out the path to dex-assets by traversing up directories similar to what we do here: https://github.com/pachyderm/pachyderm/blob/master/src/internal/minikubetestenv/deploy.go#L88-L101 ?

Reply:
Yeah that could work!